cdc/processor/sinkmanager/table_progress_heap.go

```diff
@@ -45,7 +47,14 @@ func (p *progressHeap) Len() int {
 }
 
 func (p *progressHeap) Less(i, j int) bool {
-	return p.heap[i].nextLowerBoundPos.Compare(p.heap[j].nextLowerBoundPos) == -1
+	a := p.heap[i]
```

Comment:
what will happen if two tables are both in removed state?

Reply:
Use ts to compare. Smaller higher priority.